examples/nlp/language_modeling/conf/megatron_retro_config.yaml

```diff
@@ -0,0 +1,142 @@
+name: megatron_retro
```

Comment:
let's call this something other than megatron_retro. tiny_retro or test_retro, etc. since this is only for 2 gpus.

Reply:
changed the name